chapters/my_first_jina_app/index.md

```diff
@@ -0,0 +1,389 @@
+
+### Create a new app
+
+```sh
+jina hub new --type=app
+```
+
+We recommend the following settings:
+
+| Parameter                   | What to type                                          |
+| ---                         | ---                                                   |
+| `project_name`              | `Wikipedia sentence search`                           |
+| `jina_version`              | Use default setting                                   |
+| `project_slug`              | Use default setting                                   |
+| `author_name`               | Your name                                             |
+| `project_short_description` | `Search Wikipedia sentences using Jina neural search` |
+| `task_type`                 | `2` (NLP)                                             |
+| `index_type`                | `2` (strings)                                         |
+| `public_port`               | Use default setting                                   |
+| `parallel`                  | Use default setting                                   |
+| `shards`                    | Use default setting                                   |
+| `version`                   | Use default setting                                   |
+
+After you have answered all the questions, the wizard will create [a folder and files](https://docs.google.com/document/d/1A2t-K2DEZvJ7sOmWLuCmUQINyESaxucM7wOJCSCPmdI/edit#) for your new Jina app.
+
+### Install app requirements
+
+In the terminal:
+
+```sh
+cd wikipedia_sentence_search
+pip install -r requirements.txt
+```
+
+### Download data (optional)
+
+Our goal is to search a set of sentences from Wikipedia and return the closest sentences to our search term. We use [this dataset](https://www.kaggle.com/mikeortman/wikipedia-sentences) from Kaggle.
+
+By default we just include a [subset of this data](/data/toy-input.txt), so you don't need to download anything. However, if you'd like to work with more sentences:
+
+1. Set up [Kaggle](https://www.kaggle.com/docs/api#getting-started-installation-&-authentication)
+2. `wget https://raw.githubusercontent.com/jina-ai/examples/master/wikipedia-sentences/get_data.sh`
+3. `sh ./get_data.sh`
+
+The `get_data.sh` script:
+
+- Creates a data directory
+- Downloads the dataset from Kaggle
+- Extracts and shuffles the dataset to ensure variety in what we ask Jina to search
+
+Since `app.py` indexes `data/toy-input.txt` by default, we override it with an environment variable
+
+```sh
+export JINA_DATA_FILE='data/input.txt'
+```
+
+Double check it was set successfully by running:
+
+```sh
+echo $JINA_DATA_FILE
+```
+
+## 🏃 Run the app
+
+In section you will index and search through your data
+
+### Index Flow
+
+First up we need to build up an index of our dataset, which we'll later search with our search Flow.
+
+```sh
+python app.py -t index
```

Comment:
maybe you can give a short explanation on `-t` refers to `task`

Reply:
Good catch. I moved this to first time we use `-t`